scripts/update_README.md

```diff
@@ -7,7 +7,7 @@ The update script is to be used by WSO2 subscription users such that products pa
 * [WUM](https://wso2.com/updates/wum) updated packs should be provided in the `/files/packs` directory
 
 `
-NOTE: Provided WUM updated packs should contain the latest updates for wso2is-5.8.0 and wso2is-analytics-5.8.0
+NOTE: Provided WUM updated packs should contain the latest updates for wso2is-5.9.0 and wso2is-analytics-5.8.0
```

Comment:
Shall we remove all points in the documentation saying that we need WUM updated packs? This was mentioned earlier as the in-place tool was not delivered with the GA pack and it can only be obtained through an update. However, the tool will be made available in the GA pack for Identity Server 5.9.0.

Reply:
@Savidude I think we have to leave this as it is since, the WSO2 Identity Server Analytics version 5.9.0 has not been released, yet. Version 5.8.0 GA does not ship the In-Place Update tool.

Reply:
+1 Let's make the changes when Analytics 5.9.0 is released